bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,111 @@
+
+## Channels
+
+Channels are auto-discovered, you will find them on the RVS things.
+They are organized the same way as the LCD screen of your PAC device, by top level menu functionality, and sub-functionalities.
+Each channel is strongly typed, so for example, for heating mode, openHAB will provide you with a list of choices supported by the device.
+
+Channel                | Description                                              | Type          | Unit     | Security Access Level   |  ReadOnly | Advanced
+-----------------------|----------------------------------------------------------|---------------|----------|-------------------------|-----------|-------------------
+controlBoilerApproval  | Set Boiler Approval (`AUTO`, `OFF`, `ON`)                | String        |          |                         |  R/W      | true
+controlProgram         | Set Program (`OFF`, `NORMAL`, `WARMWATER`, `MANUAL`)     | String        |          |                         |  R/W      | true
+
+Channel Type ID        | Item Type                                | Description
+-----------------------|------------------------------------------|----------------------------------------------
+Numeric                | Number                                   | Handle basic numeric value
+String                 | String                                   | a String
+TimeOfDay              | TimeOfDay                                |
+Datetime               | Datetime                                 |
+RadioButton            | RadioButton                              |
+Scheduler              | Scheduler                                |
+Temperature            | Number                                   | Use to handle reading of  temperatur
+Setpoint               | Number                                   | Handle the setting of a temperature
+Regime                 | Number                                   | Enumeration for handling mode change
+   
+## Full Example
+
+Things file `.things`
+
+```java
+Bridge siemenshvac:ozw672:local "Ozw672" [ baseUrl="https://192.168.254.42/", userName="Administrator", userPassword="mypass"  ] 
+{
+    Thing RVS41_813_327 local "RVS41.813/327"  [  ]
+    {
+        Channels:
+            Type Setpoint:temperature                 "Temperature"  [ id="1726" ]
+            Type Regime:cc1                           "CC1"          [ id="1725" ]
+    
+    }
+```
+
+Items file `.items`
+
+```java
+String              Boiler_State_Pump_HWS       "HWS Pump State [%s]"                   { channel = "siemenshvac:RVS41_813_327:local:local:2237#2259_PpeChargeECS"              }       
+Number              Boiler_State_HWS            "HWS State [%s]"                        { channel = "siemenshvac:RVS41_813_327:local:local:2032#2035_Etat_ECS"                  }
+Number:Temperature  Flow_Temperature_Real       "Flow Temparature Real [%.1f °C]"       { channel = "siemenshvac:RVS41_813_327:local:local:2237#2248_ValReelleTempDep_CC1"      }   
```

Comment:
I can't seem to find `Number:Temperature` being generated as item type in the code, perhaps you can guide me? This would require updating by `QuantityType`.

Reply:
I will have to check, I'm not sure I handle it in the code has Number:Temperature, but as simple Number.

Reply:
Fixed by UoM chnages?

Reply:
Yes, fix by UoM changes